src/app/viam-transport.ts

```diff
@@ -1,4 +1,91 @@
 import { grpc } from '@improbable-eng/grpc-web';
```

Comment:
The diff looks slightly bigger than what actually changed since this logic was also moved from [./src/robot/dial.ts](https://github.com/viamrobotics/viam-typescript-sdk/pull/202/files#diff-52a50b491f74e91af65fba14f575d59124ae522a5fb352e0185c7cfeedd3cd82). 

The main functional change is that users can now provide either an access token for direct access or a credential to be exchanged for an access token. Only the latter was accepted before.